handshake/crypto_setup_server.go

```diff
@@ -28,6 +29,7 @@ type cryptoSetupServer struct {
 	scfg                 *ServerConfig
 	stkGenerator         *STKGenerator
 	diversificationNonce []byte
+	QuicTracer           qtrace.Tracer
```

Comment:
I think this should be a pointer as in scfg above, to not copy the tracer over and over